### PR TITLE
Fix CI for php 8

### PR DESCRIPTION
**Pull request by derrabus**
Our tests currently don't work on php 8. I'd like to give it a try to fix this.

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
You'll find some thoughts at #247

**Comment by derrabus** [comment]
@nicolas-grekas I‘d like to get to the point that we can actually solve #247. Right now, the nightly run ist just plain dead. :-/

**Comment by derrabus** [comment]
Let's wait for #303.

**Comment by derrabus** [comment]
I've rebased the PR on #303.

**Comment by derrabus** [comment]
Current state of the PR: The tests should run on php 8 with PHPUnit 9.4. They pass on php 7.4 and below and all failures on php 8 should be specific to php 8. I'd suggest to merge it (after #303) to get a proper CI for the PRs that fix PHP 8 compatibility.

File: .travis.yml [hunk 1]
```diff
@@ -36,16 +28,16 @@ before_install:
...
 
 install:
     - export COMPOSER_ROOT_VERSION=dev-main
     - composer --prefer-source install
+    - if [[ $SYMFONY_PHPUNIT_VERSION = 9.* ]]; then export SYMFONY_PHPUNIT_CONFIG=phpunit-9.xml.dist; else export SYMFONY_PHPUNIT_CONFIG=phpunit.xml.dist; fi;
     - if [[ $TRAVIS_PHP_VERSION = nightly ]]; then phpenv global 7.1; ./vendor/bin/simple-phpunit install; phpenv global nightly; fi
 
 script:
-    - ./vendor/bin/simple-phpunit
+    - ./vendor/bin/simple-phpunit  -c $SYMFONY_PHPUNIT_CONFIG
```
Comment (derrabus) [inline .travis.yml:43]: I'm including a different config file on PHPUnit 9 to disable the `TestListener`. See my comment here: https://github.com/symfony/symfony/issues/33331#issuecomment-713703230

**Comment by derrabus** [comment]
🤞🏻 The listener should work now.

**Comment by nicolas-grekas** [comment]
Thank you @derrabus.

**Pull request merged**